### PR TITLE
docs: Update NOTIFICATIONS.md with detailed configuration and template information

### PR DESCRIPTION
**Pull request by jungwire**
### Description:

Updated and added additional information for the email template used for the notifications

### Checklist:

_Make sure you tick all the boxes below if they are true or do not apply before you ask for review_

- [x] I have performed a self-review of my code
- [x] I have made my code as simple as possible
- [x] I have added relevant tests for my changes and the code coverage has not dropped substantially
- [x] I have removed all commented code
- [x] I have updated the documentation in all relevant places (Javadoc, Swagger, MDs...)
- [x] I have described the PR and added a meaningful title in the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format

### CODE REVIEW

File: docs/NOTIFICATIONS.md [hunk 1]
```diff
@@ -22,7 +22,34 @@ SPRING_MAIL_PASSWORD="pass"
...
+ - `negotiations`: List of negotiation Ids.
+ - `roleForNegotiation`: List of roles for each negotiation.
+ - `titleForNegotiation`: List of titles for each negotiation.
+ - `notificationsForNegotiation`: List of notifications for each negotiation.
+
+#### Updating Templates at Runtime
+Thymeleaf supports runtime updates to email templates. This allows you to make changes without restarting the application, which is useful for quick updates or tweaks:
+ - **Enable Template Cache Refresh:** By default, Thymeleaf caches templates for performance. To enable runtime updates, you need to configure Thymeleaf to refresh templates automatically. This can typically be done by setting the `spring.thymeleaf.cache` property to `false` in your application configuration file (`application.yml`):
+```yaml
+spring.thymeleaf.cache=false
```
Comment (RadovanTomik) [inline docs/NOTIFICATIONS.md:49]: maybe set this property for application-dev.yaml? so devs can take advantage of it?

File: docs/NOTIFICATIONS.md [hunk 2]
```diff
@@ -22,7 +22,34 @@ SPRING_MAIL_PASSWORD="pass"
...
 ### Email template
 Templates ensure consistent and automated communications for various functions. 
-We are using [Thymeleaf](https://www.thymeleaf.org/) as the template engine, the email templates are stored in the `resources/templates/` directory.
+We are using [Thymeleaf](https://www.thymeleaf.org/) as the template engine, which allows us to create dynamic and customizable email content.
+
+#### Template location
+The email template is stored in the `src/main/resources/templates/` directory. It is written in HTML with Thymeleaf syntax, allowing you to include dynamic content using expressions, conditionals, and loops.
+
+#### Editing the template
+ - To edit the template, open the `email-notification.html` file.
```
Comment (RadovanTomik) [inline docs/NOTIFICATIONS.md:31]: maybe also provide instructions on how to upload a custom file for docker containers?

**Review by RadovanTomik — CHANGES_REQUESTED** [review]
please see comments, also is there a reason why it is not part of #442 ?

**Comment by jungwire** [comment]
merged this into https://github.com/BBMRI-ERIC/negotiator/tree/feat/add_variable_email_template branche and added requested changes

**Pull request closed without merging**